src/telemetry.ts

```diff
@@ -65,7 +67,16 @@ export class TelemetryOutputChannel implements vscode.OutputChannel {
       if (this.isNeedToSkip(value)) {
         return;
       }
-      this.telemetry.send({ name: 'yaml.server.error', properties: { error: this.createErrorMessage(value) } });
+      if (!this.errors) {
+        this.errors = [];
+      }
+      if (this.throttleTimeout) {
+        clearTimeout(this.throttleTimeout);
+      }
+      this.errors.push(value);
+      this.throttleTimeout = setTimeout(() => {
+        this.telemetry.send({ name: 'yaml.server.error', properties: { error: this.createErrorMessage() } });
+      }, 500);
```

Comment:
500ms is a really long time for a CPU. Maybe reduce to 50ms. You only want to eliminate duplicate errors